ca/certificate-authority.go

```diff
@@ -368,7 +378,9 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 		},
 		Serial: serialBigInt,
 	}
-
+	if !ca.forceCNFromSAN {
```

Comment:
Do we always want to do this or only when the CN is empty? (I think it's probably okay to but just wondering)